### PR TITLE
Add Google Analytics to Collections pages

### PR DESCRIPTION
**Pull request by davidrans**
I copied the GA config from `UI/templates/google_analytics_create.phtml` in the main repo.

This leaves out a couple configuration options: `userid`, `loggedIn`, `customerType` cause Jeff Snyder doesn't use them and doesn't think they're necessary for launch.

QA
---
With the production config the GA JS should show up in the `<head>` on every page and mostly match the JS on the old product list pages.

Closes https://github.com/iFixit/react-commerce/issues/483

### CODE REVIEW

File: frontend/components/analytics/GoogleAnalytics.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { GA_URL, GA_KEY } from '@config/env';
+import Head from 'next/head';
+
+export function GoogleAnalytics() {
+   return GA_URL ? (
+      <Head>
+         <script
+            key="matomo-analytics"
```
Comment (davidrans) [inline frontend/components/analytics/GoogleAnalytics.tsx:8]:
```suggestion
            key="google-analytics"
```

File: frontend/components/analytics/GoogleAnalytics.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+export function GoogleAnalytics() {
+   return GA_URL ? (
+      <Head>
+         <script
+            dangerouslySetInnerHTML={{
+               __html: `
+                  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+                  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+                  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+                  })(window,document,'script', '${GA_URL}' ?>','ga');
```
Comment (batbattur) [inline frontend/components/analytics/GoogleAnalytics.tsx:13]: Is `?>'` required here or extra left over stuff?

```suggestion
                  })(window,document,'script', '${GA_URL}','ga');
```

Comment (batbattur) [reply frontend/components/analytics/GoogleAnalytics.tsx:13]: Looks like it. This was before removing ` ?>'`:
image

This is after removing ` ?>'`:
image

**Review by andyg0808 — APPROVED** [review]

**Comment by andyg0808** [comment]
CR :dog: through 4adb2a1

**Review by batbattur — APPROVED** [review]
CR 🚛

**Pull request merged**